web/src/components/customer/listing-details/index.tsx

```diff
@@ -62,16 +64,19 @@ const ListingDetailsPage: React.FC = () => {
   return (
     <CommitFeedbackPromptContext>
       <CommitContext listingId={listingId}>
-        <ListingDetailsContext listingId={listingId}>
-          <PageContainer>
-            <BackButton pos="absolute" onClick={handleBack} />
-            <CommitsBadge pos="absolute" />
-            <ContentContainer>
-              <ListingDetails />
-            </ContentContainer>
-            <ActionBar />
-          </PageContainer>
-        </ListingDetailsContext>
+        <DeliveryDetailsPromptProvider>
```

Comment:
should you use DeliveryDetailsPromptContext here?

Reply:
@ifhuang  It is used in `DeliveryDetailsPrompt` and `ActionBar` so `DeliveryDetailsPromptProvider` has to be ancestor of those 2 react components. Where would you prefer it to be placed?